File: CodeChef/Beginner/07LuckyFour/luckyfour.java
import java.util.Scanner;

class luckyfour {
    public static int Count(int x) {
        int a, count = 0;
        while (x != 0) {
            a = x % 10;
            if (a == 4) {
                count++;
            }
            x /= 10;
        }
        return count;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int x, n;
        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            System.out.println(Count(x));
        }
        sc.close();
    }
}

File: CodeChef/Beginner/07LuckyFour/luckyfour.cpp
#include <iostream>
using namespace std;

class luckyfour {
public:
  int count(int x) {
    int count = 0, a;
    while (x != 0) {
      a = x % 10;
      if (a == 4) {
        count += 1;
      }
      x /= 10;
    }

    return count;
  }
};
int main() {
  int n, x;
  cin >> n;
  luckyfour lfobj = luckyfour();
  for (int i = 0; i < n; i++) {
    cin >> x;
    cout << lfobj.count(x) << endl;
  }
}